### PR TITLE
Remove onboarding

### PR DESCRIPTION
**Pull request by jcapona**
| Status  | Ticket/Issue |
| :---: | :--: |
| Ready/Hold | [Ticket](https://pi-top.atlassian.net/browse/SWE-381) |

##### Summary
- On first boot, pi-top[4] opens landing app and legacy devices open onboarding in a window.
- Replaced final page of onboarding with a 'finish' page that on completion takes user to landing.

##### Detailed changes

Backend:
- `/` route now doesn't check if the device is onboarded or not. It used to do this to redirect users to onboarding when not complete.
- Miniscreen onboarding will not start anymore if device is not onboarded. The app files are still in the repo since they might be useful in the future. 
- Updated `pt_os_web_portal/app_window/__main__.py`, used to start our apps in the devices. 
  - Updated so that when called without an argument, it will open the 'landing' if the device running the script is a pi-top[4] or 'os-setup' (onboarding) on legacy devices.
  - Updated onboarding app so that it opens in a window (class `OnboardingAppWindow`). Check attached screenshots.
- On first boot, we use a `.desktop` file to open the onboarding or landing page.
  - Added `pt-first-boot-app.desktop` which will open the onboarding or landing depending on the device type.
  - Removed `pt-os-setup.desktop`.
  - Removed `pt-os-landing.desktop`
- Add `/close-first-boot-app-window` route that closes the 'first time boot' app window in the desktop.
- Updated backend routes and methods that would handle these `.desktop` files.
  - Removed `/disable-landing` route and `disable_landing` method since they will not be used anymore.
  - Removed `/disable-onboarding` route, not used anymore.
  - Removed `/configure-landing` route and `configure_landing` method since they will not be used anymore.
  - Don't remove `pt-os-setup.desktop` file in `deprioritise_openbox_session` functon. 
  - Added `/stop-first-boot-app-autostart` route to handle removing `pt-first-boot-app.desktop`.
 
Frontend:
- Updated services according to changes in routes to focus on 'first boot app' instead of on 'landing' and 'onboarding'
- Removed `restartPage` component.
- Added `finalOnboardingPage` (Check attached screenshot).
- When finished, the onboarding now redirects to landing page.
- Created `closeButton` component, extracted from the landing page. It's now used in the onboarding & landing when running the app in the device.

#### Screenshots (feature, test output, profiling, dev tools etc)

new 'final' page:

image

onboarding on legacy devices:

image

#### Other notes (e.g. implementation quirks, edge cases, questions / issues)
-

#### Manual testing tips
-

#### Tag anyone who definitely needs to review or help
-

### CODE REVIEW

File: frontend/src/components/header/Header.tsx [hunk 1]
```diff
@@ -8,6 +13,19 @@ export default () => {
       <div className={styles.logoContainer}>
           <div className={styles.logo} />
       </div>
+      {runningOnWebRenderer() &&
+        <MaskedDiv
```
Comment (angusjfw) [inline frontend/src/components/header/Header.tsx:17]: i think you meant to use <CloseButton here

File: frontend/src/components/closeButton/CloseButton.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+type Props = {
+  onClose?: () => void;
+  className?: string;
+};
+
+export default ({ onClose, className }: Props) => {
+  return (
+    <MaskedDiv
+      className={cx(styles.closeButtonDiv, className)}
+      mask={`url(${closeButtonImage})`}
```
Comment (angusjfw) [inline frontend/src/components/closeButton/CloseButton.tsx:17]: if there is a circle background on this we can make visible i think that would look better. not super important by maybe it's here and transparent

**Review by angusjfw — APPROVED** [review]
